plugins/idea/pluginBase/src/main/java/com/imaginea/kodebeagle/base/settings/ui/SettingsPanel.java

```diff
@@ -84,15 +81,15 @@ public final IdentityPanel getIdentityPanel() {
     }
 
     private final ImportsPanel importsPanel = new ImportsPanel();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)